### PR TITLE
Clean render buffers before writing

### PR DESCRIPTION
**Pull request by MyBlackMIDIScore**
Was a thing in VoiceChannel but not in ChannelGroup or BufferedRenderer and it created confusion.

I think this is the last thing I wanted to do before release. If I remember anything else I'll put it in this PR.

### CODE REVIEW

**Review by arduano — APPROVED** [review]
Performance should be negligible I assume

**Comment by MyBlackMIDIScore** [comment]
> Performance should be negligible I assume

I didn't really test this tbh, but I'll run the core benchmarks before I merge. But anyway if this isn't done by the synth itself, it will be done by the program using it so there shouldn't be a difference at the end.

I'll merge this later today (after I do the benchmarks too) in case there's anything else to fix before release.

**Comment by arduano** [comment]
Just side note, respond to my email probably, if I sent it to the right email address. Just trying to find a way to contact you that's not public like github issues.

**Comment by MyBlackMIDIScore** [comment]
> Just side note, respond to my email probably, if I sent it to the right email address. Just trying to find a way to contact you that's not public like github issues.

I haven't received anything in my inbox. I'll send you an email to the one you use in your git commits.

**Comment by MyBlackMIDIScore** [comment]
Just ran the `render` bench and results were pretty much the same.
Merging this and then I'll create the 0.1.0 tag and release.

**Pull request merged**
